plenum/recorder/src/simple_zstack_with_recorder.py

```diff
@@ -0,0 +1,47 @@
+import os
+from typing import Set
+
+from stp_core.common.log import getlogger
+
+from plenum.recorder.src.recorder import Recorder
+from storage.kv_store_leveldb_int_keys import KeyValueStorageLeveldbIntKeys
+from stp_zmq.simple_zstack import SimpleZStack
+
+logger = getlogger()
+
+
+class SimpleZStackWithRecorder(SimpleZStack):
```

Comment:
Why can't we inherit from `KITZStack` to avoid isinstance checks?

Reply:
Because clientstack does not extend `KITZStack`